### PR TITLE
adding assets,color,fonts,component

### PR DESCRIPTION
**Pull request by JehadEAlomour**

### CODE REVIEW

File: lib/core/component/CardView.dart [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Column CardView(
+    {required String src,
```
Comment (MohammedMansour4009) [inline lib/core/component/CardView.dart:5]: please reformat code here

File: lib/core/component/CardView.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                blurRadius: 4,
+                // Blur radius of the shadow
+                offset: Offset(0, 2), // Offset of the shadow
+              ),
+            ],
+          ),
+          child: Image(
+            width: 98,
+            height: 98,
+            image: AssetImage(src),
```
Comment (MohammedMansour4009) [inline lib/core/component/CardView.dart:33]: please change the name src to imageSrc

File: assets/home_bc.png
[the remark this answers is not in the document]
Comment (JehadEAlomour) [reply assets/home_bc.png:1]: done

File: lib/core/component/CardView.dart [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      ),
+      Container(
+        alignment: Alignment.center,
+        child: Text(
+          text,
+          style: const TextStyle(
+              color: Colors.black,
+              fontSize: 16,
+              fontWeight: FontWeight.bold,
+              fontFamily: "Cairo"),
```
Comment (MohammedMansour4009) [inline lib/core/component/CardView.dart:46]: Cairo put fontFamily in constant class

File: lib/core/component/CompanyProfileCard.dart [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+
+Positioned CompanyProfilePage(
```
Comment (MohammedMansour4009) [inline lib/core/component/CompanyProfileCard.dart:3]: change name it

File: lib/core/component/TextLabel.dart [hunk 5]
```diff
@@ -3,4 +3,4 @@ import 'package:flutter/cupertino.dart';
 
 Text SmaleText(String text) => Text(text, style: TextStyle(fontSize: 12));
 
-Text NormalText(String text, {double? fontSize, FontWeight? fontWeight}) => Text(text,style: TextStyle(fontSize: fontSize ,fontWeight: fontWeight));
+Text NormalText(String text, {double? fontSize, FontWeight? fontWeight}) => Text(text,style: TextStyle(fontSize: fontSize ,fontWeight: fontWeight,fontFamily: 'Cairo'));
```
Comment (MohammedMansour4009) [inline lib/core/component/TextLabel.dart:6]: Also  here " Cairo"

File: lib/core/component/button.dart [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+import '../constant/color.dart';
+import 'TextLabel.dart';
+
+Container elevatedButton({required String text, VoidCallback? onPressed}) {
+  return Container(
```
Comment (MohammedMansour4009) [inline lib/core/component/button.dart:7]: حط اسم يعبر يمون زي الاسم الي بالجيما

**Pull request merged**